api/src/couchdb/notebooks.ts

```diff
@@ -566,39 +562,6 @@ export const validateNotebookID = async (
   return false;
 };
 
-/**
- * getNotebookRecords - retrieve all data records for this notebook
- * including record metadata, data fields and annotations
- * @param project_id project identifier
- * @returns an array of records
- */
-export const getNotebookRecords = async (
-  project_id: string,
-  token: TokenContents
-): Promise<any | null> => {
-  const records = await getRecordsWithRegex(token, project_id, '.*', true);
-  const fullRecords: any[] = [];
-  for (let i = 0; i < records.length; i++) {
-    const data = await getFullRecordData(
-      project_id,
-      records[i].record_id,
-      records[i].revision_id,
-      true
-    );
-    fullRecords.push(data);
-  }
-  return fullRecords;
-};
-
-const getRecordHRID = (record: any) => {
-  for (const possible_name of Object.keys(record.data)) {
-    if (possible_name.startsWith(HRID_STRING)) {
-      return record.data[possible_name];
-    }
-  }
-  return record.record_id;
-};
```

Comment:
I see you removed - okay - reasoning?

Reply:
It was only used in this module, getRecordsWithRegex now returns full records, when I wrote this it didn't so I had to go fetch every record separately.